frontend/integration-tests/views/cluster-settings.view.ts

```diff
@@ -1,5 +1,19 @@
-import { element, by, browser, $$ } from 'protractor';
+import { element, by, browser, $$, $ } from 'protractor';
 import { waitForNone } from '../protractor.conf';
 
 export const heading = element(by.cssContainingText('h1.co-m-pane__heading', 'Cluster Settings'));
 export const isLoaded = async() => await browser.wait(waitForNone($$('.co-m-loader')));
+export const channelUpdateLink = $('[data-test-id="current-channel-update-link"]');
+export const channelDropdownButton = $('[data-test-id="dropdown-button"]');
+export const getSelectedChannel = $$('[data-test-id="dropdown-menu"]');
+export const channelPopupCancelButton = $('[data-test-id="cancel-action"]');
+export const viewSchemaButton = $('[data-test-id="view-schema"]');
+export const globalConfigActionsMenu = $('[data-test-id="actions-menu-button"]');
+export const globalConfigEditModelInput = $('[data-test-id="tags-input"]');
+export const detailsCloseButton = $('[data-test-id="close"]');
+export const globalConfigEditLabels = $('[data-test-action="Edit Labels"]');
+export const globalConfigResourceRow = $('[data-test-action="Console"]').$$('[data-test-id="kebab-button"]');
+export const resourceLink = (name) => $(`[data-test-id=${name}]`);
+export const viewDetailsButton = (name)=> $(`[data-test-id=${name}]`);
+export const globalConfigKebabDropDownButton = (name)=> $(`[data-test-action="${name}"]`);
+export const clusterDetailsHeading = (name) => $(`[data-test-id=${name}]`);
```

Comment:
If there are only a handful of each, I'd just have constants for the ones we know about. For instance,

```js
export const globalConfigHeading = $('[data-test-id="global-config-resource-header"]');
```